taxi/views.py

```diff
@@ -87,3 +102,19 @@
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(generic.CreateView):
```

Comment:
Consider adding `LoginRequiredMixin` to `DriverCreateView` to ensure that only authenticated users can create drivers.